<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Equalizer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1">
    <item>
     <layout class="QHBoxLayout" name="controlsLayout">
      <property name="spacing">
       <number>12</number>
      </property>
      <item>
       <widget class="QLabel" name="labelPreset">
        <property name="text">
         <string>Preset:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignVCenter|Qt::AlignLeft</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="presetComboBox"/>
      </item>
      <item>
       <widget class="QCheckBox" name="bypassCheckBox">
        <property name="text">
         <string>Bypass</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="resetButton">
        <property name="text">
         <string>Reset</string>
        </property>
        <property name="toolTip">
         <string>Reset all bands to 0 dB</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="EqualizerWidget" name="equalizerWidget"/>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>EqualizerWidget</class>
   <extends>QWidget</extends>
   <header>EqualizerWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
